linux57-tkg/linux57-tkg-config/prepare

```diff
@@ -1,68 +1,229 @@
-  patch -Np1 -i ../enable_additional_cpu_optimizations_for_gcc_v10.1%2B_kernel_v5.7%2B.patch
+  if [ "${_distro}" == "Arch" ]; then
+    patch -Np1 -i "$_where"/enable_additional_cpu_optimizations_for_gcc_v10.1%2B_kernel_v5.7%2B.patch
+  else
+    patch -Np1 -i "$_where"/enable_additional_cpu_optimizations_for_gcc_v10.1+_kernel_v5.7+.patch
+  fi
+  msg2 "Done"
 
   # TkG
   msg2 "Applying clear linux patches"
-  patch -Np1 -i ../0002-clear-patches.patch
+  patch -Np1 -i "$_where"/0002-clear-patches.patch
+  msg2 "Done"
 
   msg2 "Applying glitched base patch"
-  patch -Np1 -i ../0003-glitched-base.patch
+  patch -Np1 -i "$_where"/0003-glitched-base.patch
+  msg2 "Done"
 
-  if [ "$_misc_adds" = "true" ]; then
+  if [ "$_distro" == "Arch" ]; then
     msg2 "Applying misc additions patch"
-    patch -Np1 -i ../0012-misc-additions.patch
+    patch -Np1 -i "$_where"/0012-misc-additions.patch
   fi
 
   if [ "${_cpusched}" == "MuQSS" ]; then
     # MuQSS
-    patch -Np1 -i ../0004-5.7-ck1.patch
+    msg2 "Applying MuQSS base patch"
+    patch -Np1 -i "$_where"/0004-5.7-ck1.patch
+    msg2 "Done"
+    
     if [ "${_aggressive_ondemand}" == "true" ]; then
-      patch -Np1 -i ../0004-glitched-ondemand-muqss.patch
+      msg2 "Applying MuQSS agressive ondemand governor patch"
+      patch -Np1 -i "$_where"/0004-glitched-ondemand-muqss.patch
+      msg2 "Done"
     fi
-    patch -Np1 -i ../0004-glitched-muqss.patch
+
+    msg2 "Applying Glitched MuQSS patch"
+    patch -Np1 -i "$_where"/0004-glitched-muqss.patch
+    msg2 "Done"
+  
   elif [ "${_cpusched}" == "pds" ]; then
     # PDS-mq
-    patch -Np1 -i ../0005-v5.7_undead-pds099o.patch
+    msg2 "Applying PDS base patch"
+    patch -Np1 -i "$_where"/0005-v5.7_undead-pds099o.patch
+    msg2 "Done"
+
     if [ "${_aggressive_ondemand}" == "true" ]; then
-      patch -Np1 -i ../0005-glitched-ondemand-pds.patch
+      msg2 "Applying PDS agressive ondemand governor patch"
+      patch -Np1 -i "$_where"/0005-glitched-ondemand-pds.patch
+      msg2 "Done"
     fi
-    patch -Np1 -i ../0005-glitched-pds.patch
+
+    msg2 "Applying Glitched PDS patch"
+    patch -Np1 -i "$_where"/0005-glitched-pds.patch
+    msg2 "Done"
+  
   elif [ "${_cpusched}" == "bmq" ]; then
+    
     # Project C / BMQ
+    msg2 "Applying Project C / BMQ base patch"
+    
     patch -Np1 -i ../0009-prjc_v5.7-r3.patch
+    msg2 "Done"
+
     if [ "${_aggressive_ondemand}" == "true" ]; then
-      patch -Np1 -i ../0009-glitched-ondemand-bmq.patch
+      msg2 "Applying BMQ agressive ondemand governor patch"
+      patch -Np1 -i "$_where"/0009-glitched-ondemand-bmq.patch
+      msg2 "Done"
     fi
-    patch -Np1 -i ../0009-glitched-bmq.patch
+
+    msg2 "Applying Glitched BMQ patch"
+    patch -Np1 -i "$_where"/0009-glitched-bmq.patch
+    msg2 "Done"
   else
-    patch -Np1 -i ../0003-glitched-cfs.patch
+    msg2 "Applying Glitched CFS patch"
+    patch -Np1 -i "$_where"/0003-glitched-cfs.patch
+    msg2 "Done"
```

Comment:
I'm not convinced by those "Done" messages. Am I missing something?

Reply:
I was debugging in the beginning, and I wanted to know which lines are terminating properly. If you don't like the "Done" let's get rid of it!

Reply:
I haven't actually ran the script to see the output yet, but with the patch coming after outputting it's "header message" I gave you my initial thoughts about it. I'm not against them if you consider they are helping out.

Reply:
I find them useful and like to see that something is "Done" before it moves to something else. The final decision is up to you, it will take me few seconds to remove all the "Done" messages.